src/runtime/exec_context.ts

```diff
@@ -288,13 +292,14 @@ export default class ExecContext<M> {
     const { container, env } = this;
     return (
       !container.subscriptions && [] ||
+      // @TODO Filter out empty values, like how commands work
```

Comment:
Also, if you could scout this, I would be eternally grateful. At least for a while.